guides/release/testing/different-types-of-tests.md

```diff
@@ -0,0 +1,221 @@
+Ember provides 3 types of tests out of the box:
```

Comment:
Looking good, thanks for the updates.  I made another pass and added a few more small comments.  After that I'll solicit another learning team member to take a pass and we can merge.

Reply:
Thanks again very much. I enjoyed working with you and learned a lot from your feedback!